owncloudData/src/main/java/com/owncloud/android/data/spaces/datasources/implementation/OCLocalSpacesDataSource.kt

```diff
@@ -77,11 +77,11 @@ class OCLocalSpacesDataSource(
                 id = space.id,
                 lastModifiedDateTime = space.lastModifiedDateTime,
                 name = space.name,
-                owner = SpaceOwner(
-                    user = SpaceUser(
-                        id = space.ownerId
+                owner = space.ownerId?.let { spaceOwnerIdEntity ->
+                    SpaceOwner(
+                        user = SpaceUser(id = spaceOwnerIdEntity)
```

Comment:
I would put this in different lines as in the other methods to improve readability
 ```suggestion
                         user = SpaceUser(
                             id = spaceOwnerIdEntity
                         )
```

Reply:
Since it's just one attribute, I added it to the same line, but can change it, np